src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = new Big(0);
+  const x = new Big(numberOne);
+  const y = new Big(numberTwo);
+  switch (operation) {
+    case '+':
+      result = x.plus(y);
+      break;
+    case '-':
+      result = x.minus(y);
+      break;
+    case '*':
+      result = x.times(y);
+      break;
+    case '/':
+      if (y === 0) {
+        return 'Math Error';
+      }
+      result = x.div(y);
+      break;
+    case '%':
+      result = numberOne % numberTwo;
```

Comment:
The % operator should return the number / 100  ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)